packages/suite/src/components/wallet/CoinmarketBuyOfferInfo/index.tsx

```diff
@@ -134,9 +133,12 @@ const CoinmarketBuyOfferInfo = ({ selectedQuote, transactionId, providers, accou
                         <Translation id="TR_BUY_BUY" />
                     </LeftColumn>
                     <RightColumn>
-                        <Dark>{`${formatCryptoAmount(
-                            Number(receiveStringAmount),
-                        )} ${receiveCurrency}`}</Dark>
+                        <Dark>
+                            <FormattedCryptoAmount
+                                value={receiveStringAmount}
+                                symbol={receiveCurrency?.toLowerCase()}
```

Comment:
`receiveCurrency` holds a value that comes from BE and is upper case. Since `toLowerCase()` is called, I think it makes it obvious that lower case value is required.

Reply:
I mean, in general, the `symbol` prop should be case-agnostic ;) so perhaps you can add it to this thing 

```
const { features: networkFeatures, testnet: isTestnet } =
        NETWORKS.find(network => network.symbol === symbol.toLowerCase()) ?? {};
```

Reply:
Oh, I misinterpreted your first comment, sorry. This is what I wanted to do when we first talked about the component so I'm happy to fix now.